contrib/patterns/_grid-push.scss

```diff
@@ -1,7 +1,7 @@
-.grid-push--3 {
-  @include grid-push(3);
+.grid-shift--4 {
```

Comment:
Wait…did you mean to make these changes in this partial?

Are they a dupe of `_grid-shift.scss` below?